web/src/connectors/task-annotator-connector/use-split-validation.ts

```diff
@@ -78,24 +81,54 @@ export default function useSplitValidation({
         { enabled: isSplitValidation }
     );
 
-    const userPages: AnnotationsByUserObj[] = useMemo(() => {
-        if (!byUser) {
+    const { data: pages } = useGetPageSummary(
+        { taskId: task?.id, taskType: task?.is_validation },
+        { enabled: Boolean(task) }
+    );
+
+    const { data: latestRevision } = useLatestAnnotationsByUser(
+        {
+            fileId,
+            jobId: job?.id,
+            pageNumbers: pages?.annotated_pages // передать сюда annotated из page sammury
+        },
+        { enabled: isSplitValidation }
+    );
+
+    const latestRevisionByAnnotators: UserRevision[] = useMemo(() => {
+        if (!latestRevision) {
             return [];
         }
-        return byUser[currentPage].filter((userPage) => userPage.user_id !== userId);
-    }, [byUser, currentPage]);
 
-    const taxonLabels = useAnnotationsTaxons(userPages);
+        const result = convertToRevisionByUserArray(latestRevision as unknown as LatestRevisionObj);
+        const currentUser = userId;
```

Comment:
Do we really need to put `userId` to additional variable? Maybe we can just use `userId`?

Reply:
I did this because it helped me improve the readability of the code.